docs/src/model_setup/architecture.md

```diff
@@ -1,10 +1,9 @@
 # Architecture
 
-Passing `architecture = CPU()` or `architecture = GPU()` to the `Model` constructor will determine whether the model
-is time stepped on a CPU or GPU.
+Passing `CPU()` or `GPU()` to the grid constructor determines whether the grid lives on a CPU or GPU.
 
-Ideally a set up or simulation script does not need to be modified to run on a GPU but we are still smoothing out
-rough edges. Generally the CPU wants `Array` objects while the GPU wants `CuArray` objects.
+Ideally a set up or simulation script does not need to be modified to run on a GPU but still we are smoothing
+out rough edges. Generally the CPU wants `Array` objects while the GPU wants `CuArray` objects.
```

Comment:
point out that users should raise issues to improve the API for CPU->GPU transitions?

Reply:
done